configuration/pih/htmlforms/ovcFollowup.xml

```diff
@@ -0,0 +1,264 @@
+            min-width: initial;
+        }
+
+        .inline-obs > span {
+            display: inline-block;
+        }
+
+        .light-font {
+            font-family: "OpenSansLight",sans-serif;
+        }
+
+        .small-obs {
+            max-width: 10em;
+            display: block;
+            margin-bottom: 1em;
+        }
+
+        .small-obs input {
+            min-width: 4em;
+            display: inline;
+            margin-right: 0.5em;
+        }
+
+        .medium-obs {
+            max-width: 16em;
+            display: block;
+            margin-bottom: 1em;
+        }
+
+        .medium-obs input {
+            min-width: 8em;
+            margin-right: 0.5em;
+        }
+
+        .small-text {
+            color: #555555;
+            font-size: 90%;
+            display: block;
+        }
+
+        .section-container {
+            background: #F2F2F2;
+            box-shadow: 3px 3px 3px 1px rgba(0, 0, 0, 0.2);
+            padding: 10px 5px 10px 15px;
+            margin: 10px 0;
+            line-height: 1.5em; /*add this for vertical spacing between elements*/
+        }
+
+        .section-container input[type="checkbox"] {
+            margin: 0px 5px; /*changed values to vertical, horizontal*/
+            top:5px; /*added to offset the checkbox position to line up*/
+        }
+
+        .section-container label { /*new definition to override labels inside section-containers*/
+            margin: 0px;
+        }
+
+        .question-container {
+            padding: 5px 5px 10px 15px;
+            margin: 5px 0;
+            line-height: 1.5em; /*add this for vertical spacing between elements*/
+        }
+
+        .question-container label {
+            margin-top: 5px;
+        }
+
+       .section {
+         width: 95%;
+        }
+
+        legend {
+          width: auto;
+          font-size: 16px;
+        }
+
+        textarea {
+            width: 95%;
+        }
+
+        /* Grey out disabled checkboxes */
+        input[type=checkbox][disabled] {
+            filter: invert(15%);
+        }
+
+        .section-button {
+            margin: 10px 0 0 24px;
+        }
+
+        @media print {
+            .print-form-datestamps { display: block !important }
+            button.cancel, button.submit { display: none }
+            label { font-weight: bold }
+            label[for] { font-weight: normal } /* refers to the radio set label */
+        }
+
+    </style>
+
+    <div class="print-form-datestamps" style="display:none">
+        <p><uimessage code="created_on"/>:
+            <lookup complexExpression="$form.dateCreated"/>
+        </p>
+        <p><uimessage code="last_updated_on"/>:
+            <lookup complexExpression="$form.dateChanged"/>
+        </p>
+        <p><uimessage code="printed_on"/>:
+            <lookup complexExpression="$formGeneratedDatetime"/>
+        </p>
+    </div>
+
+    <includeIf velocityTest="$user.hasPrivilege('Task: emr.retroConsultNote')">
```

Comment:
It's probably fine to just have this one block for the date/location/provider instead of the three we've been using elsewhere, but then there should be no includeIf around this block... ie, we need to make sure the "who-when-where" *always* appears on the form, but it's probably fine to have this version, that allows editing all the three values, displayed for all users, instead of all the conditional stuff.  Could check with Dave or Ellen.  Also, same goes for the Intake form.

Reply:
This is much cleaner.  That section kept growing and was duplicated too many times.  Thanks @mogoodrich .  I'll approve.